python/sglang/srt/managers/schedule_batch.py

```diff
@@ -321,16 +321,54 @@ def init_new(cls, reqs, req_to_token_pool, token_to_kv_pool, tree_cache):
             return_logprob=return_logprob,
         )
 
+    def batch_size(self):
+        return len(self.reqs) if self.reqs is not None else 0
```

Comment:
Why encapsulate the previous `len(self.reqs)` in a function and include a None check? Why wasn't this problematic previously?

Reply:
No, the current implementation and the previous one are both well with `len(reqs)`.

Reply:
ok